zerver/context_processors.py

```diff
@@ -56,19 +56,19 @@ def zulip_default_context(request):
     """
     realm = get_realm_from_request(request)
 
+    realm_uri = settings.SERVER_URI
+    realm_name = None
+    realm_icon = None
+    realm_description = None
+    realm_invite_required = False
```

Comment:
Huh, or we could swap the order -- put the `None` case first.

I think I prefer that over pulling it out of the conditional altogether. Feels a little clearer how there are two cases here. I'd rather keep the `None` case first, and it seems kind of awkward that mypy accepts one but not the other, but we can take that up separately.